Makefile

```diff
@@ -55,25 +50,15 @@ ensure-juju-test: ensure-juju-env
 		|| (echo 'The "juju-test" command is missing.  See HACKING.md' \
 		; false)
 
-.PHONY: ftest
-ftest: setup ensure-juju-test
-	$(JUJUTEST) 20-functional.test
-
-# This will be eventually removed when we have juju-test --clean-state.
-.PHONY: test
-test: unittest ftest
-
-# This will be eventually renamed as test when we have juju-test --clean-state.
-.PHONY: jujutest
-jujutest:
-	$(JUJUTEST)
-
 .PHONY: lint
 lint: setup
 	@$(VENV)/bin/flake8 --show-source \
 		--exclude=.venv,charmhelpers \
-		--filename *.py,20-functional.test \
-		hooks/* tests/ server/
+		--filename *.py,config-changed,start,stop,upgrade-charm \
+		hooks/ tests/ server/
+
+.PHONY: check
+check: test lint
```

Comment:
Could you add `clean` to this target? And as lint usually goes faster and may be the source of test failures, swapping them might make sense, so `clean lint test`.